protos/flyteidl/core/tasks.proto

```diff
@@ -194,6 +194,25 @@ message DataLoadingConfig {
   // In the inputs folder, there will be an additional summary/metadata file that contains references to all files or inlined primitive values.
   // This format decides the actual encoding for the data. Refer to the encoding to understand the specifics of the contents and the encoding
   MetadataFormat format = 3;
-   // Flag enables DataLoading Config. If this is not set, data loading will not be used! 
-   bool enabled = 4;
+  // Flag enables DataLoading Config. If this is not set, data loading will not be used! 
+  bool enabled = 4;
+  // Strategy to use when dealing with Blob, Schema, or multipart blob data (large datasets)
+  enum BlobDownload {
```

Comment:
Don't we need the same for Schema type?
Should we have the setting say something like `OffloadedTypeDownload.Staregy` instead? and apply that to blobs and schemas (and potentially later to array outputs)

Reply:
Second, I'd much prefer to not have Blob in the name.